calico-cloud/visibility/elastic/flow/datatypes.mdx

```diff
@@ -64,3 +64,38 @@ The following table details the key/value pairs in the JSON blob, including thei
 | `tcp_total_retransmissions`       | long              | Total retranmitted packets. This field is only available if flowLogsEnableTcpStats is enabled                                                                                                                                                                                                                                                        |
 | `tcp_lost_packets`                | long              | Total lost packets. This field is only available if flowLogsEnableTcpStats is enabled                                                                                                                                                                                                                                                                |
 | `tcp_unrecovered_to`              | long              | Total unrecovered timeouts. This field is only available if flowLogsEnableTcpStats is enabled                                                                                                                                                                                                                                                        |
+
+### Format of the policies field
+
+The `policies` field contains a comma-delimited list of policy rules that matched the flow.  Each entry in the
+list has the following format:
+
+```
+<index>|<tier name>|<policy name>|<action>|<rule index>
+```
+
+Where,
+
+* `<index>` numbers the order in which the rules were hit, starting with `0`.
+  :::tip
+  Sort the entries of the list by the `<index>` to see the order that rules were hit.
+  :::
+
+* `<tier name>` is the name of the policy tier containing the policy, or `__PROFILE__` for a rule derived from a
+  `Profile` resource (this is the internal datatype used to represent a Kubernetes namespace and its associated
+  "default allow" rule).
+* `<policy name>` is the name of the policy/profile; its format depends on the type of policy:
+
+  * `<tier>.<name>` for {{prodname}} `GlobalNetworkPolicy`.
+  * `<namespace>/<name>` for Kubernetes `NetworkPolicy`.
+  * `<namespace>/<tier>.<name>` for {{prodname}} `NetworkPolicy`.
+
+  Staged policy names are prefixed with "staged:".
+
+* `<action>` is the action performed by the rule; one of `allow`, `deny`, `pass`.
+* `<rule index>` if non-negative, is the index of the rule that was matched within the policy, starting with 0.
+  Otherwise, a special value:
+
+  * `-1` means that the flow didn't match any policy. It hit the default action for the tier.  In this case, the
```

Comment:
I wouldn't word this as didn't match any policy.  I try and use "match" to refer to rules only, and "apply" or "select" for policies.

Something like "means the reporting endpoint was selected by the policy but no rule matched and it hit the default action for the tier. ..." maybe?  Or maybe even replace "hit the default action for the tier" with "was denied at the end of the tier", since the default action isn't configurable.

Reply:
I could be wrong, but I think the default action can be "pass" if you have a staged tier?